gst/nnstreamer/tensor_filter/tensor_filter_common.c

```diff
@@ -856,74 +983,104 @@ gst_tensor_filter_common_set_property (GstTensorFilterPrivate * priv,
         }
 
         prop->output_meta.num_tensors = num_names;
+      } else if (value) {
+        /** Update the properties */
+        if (GST_TF_FW_V0 (priv->fw)) {
+          /* Once configured, it cannot be changed in runtime */
+          g_assert (!prop->output_configured);
+        } else {
+          /** TODO: update the properties */
+        }
       }
       break;
     case PROP_CUSTOM:
-      /* In case updated custom properties in runtime! */
-      g_free_const (prop->custom_properties);
-      prop->custom_properties = g_value_dup_string (value);
+      if (!priv->prop.fw_opened) {
+        g_free_const (prop->custom_properties);
+        prop->custom_properties = g_value_dup_string (value);
+      } else {
+        /** TODO: update the properties */
```

Comment:
1. uh... why have you added this "if" statement?
2. we may need to send "event" for V1 with this event.

Reply:
1. For V0, once the framework is opened, there is no mechanism to notify the framework about the update in custom properties. Updating here directly will require each framework to perform check on updates in custom properties at every invoke (python filter uses custom_properties when the filter is opened. Directly changing custom properties will not reflect the corresponding update in the filter). Hence, for now, I have disabled updating custom properties for V0. Please advice.
2. Updated to send the event for V1.